globals-mainnet.nix

```diff
@@ -0,0 +1,27 @@
+self: super: {
+  globals = import ./globals-defaults.nix // rec {
```

Comment:
Doesn't that need `self` as argument now?

Reply:
no yet, since #23 is not merged.